src/twisted/python/test/test_setup.py

```diff
@@ -363,3 +363,39 @@ def test_find_package_modules(self):
             ]),
             sorted(result),
         )
+
+
+
+class LongDescriptionTests(SynchronousTestCase):
+    """
+    Tests for C{_getLongDescriptionArgs()}
```

Comment:
Nitpick: direct references like this are generally an `L{}`.

Reply:
:sob:

I seem to be unable to internalize epytext.